code/game/machinery/computer/abnormality_work.dm

```diff
@@ -164,6 +164,7 @@
 	if(meltdown == MELTDOWN_CYAN)
 		work_chance -= 20
 	var/work_speed = 2 SECONDS / (1 + ((get_attribute_level(user, TEMPERANCE_ATTRIBUTE) + datum_reference.understanding) / 100))
+	work_speed = work_speed / user.physiology.work_speed_mod
```

Comment:
This can be written more succinctly:
```suggestion
	work_speed /= user.physiology.work_speed_mod
```
`/=` sets the left hand side, in this case `work_speed`, to the current value of the left hand side divided by the right hand side. Compared to `left = left / right`, `left /= right` is faster and less prone to error (like order-of-operations issues once `right` becomes a large expression).